### PR TITLE
PMM-7406 IA settings tests

### PR DESCRIPTION
**Pull request by yurkovychv**

### CODE REVIEW

File: pmm-app/tests/databaseChecks_test.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Feature('Database Failed Checks');
+Feature('Database Failed Checks').retry(3);
```
Comment (puneet0191) [inline pmm-app/tests/databaseChecks_test.js:1]: why?

Comment (yurkovychv) [reply pmm-app/tests/databaseChecks_test.js]: testing it out

**Review by puneet0191 — APPROVED** [review]

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: pmm-app/codeceptConfigHelper.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    qanPagination: './tests/QAN/pages/qanPaginationFragment.js',
+    remoteInstancesPage: './tests/pages/remoteInstancesPage.js',
+    rulesAPI: './tests/ia/pages/api/rulesAPI.js',
+    ruleTemplatesPage: './tests/ia/pages/ruleTemplatesPage.js',
+    settingsAPI: './tests/pages/api/settingsAPI.js',
+    templatesAPI: './tests/ia/pages/api/templatesAPI.js',
+
+  },
+  getChunks: (files) => {
+    const conflictGroup = files.filter((value) => value.includes('PMMSettings')
```
[the remark this answers is not in the document]
Comment (yurkovychv) [reply pmm-app/codeceptConfigHelper.js]: thanks done

File: pmm-app/codeceptConfigHelper.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      || value.includes('database') || value.includes('ia') || value.includes('permissions'));
+    const otherGroup = files.filter((val) => (conflictGroup.indexOf(val) === -1));
```
[the remark this answers is not in the document]
Comment (yurkovychv) [reply pmm-app/codeceptConfigHelper.js]: much better, thanks

File: pmm-app/codeceptConfigHelper.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    return [
+      conflictGroup,
+      otherGroup,
```
[the remark this answers is not in the document]
Comment (yurkovychv) [reply pmm-app/codeceptConfigHelper.js]: renamed vars

**Review by atymchuk — APPROVED** [review]

**Pull request merged**